delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/context/ContextValue.scala

```diff
@@ -98,27 +111,31 @@ object ContextValue {
     * A remote Iri context value
     */
   final case class ContextRemoteIri(iri: Iri) extends ContextValueEntry { self =>
-    override def value: Json                                                                                         = iri.asJson
-    override val isEmpty: Boolean                                                                                    = false
-    override def merge(that: ContextValue): ContextValue                                                             =
+    def isPlatformContext: Boolean                       = iri.startsWith(contexts.base)
```

Comment:
won't this be a problem if someone creates a context with `@id` starting with `contexts.base`?

Reply:
Yes. I thought about it. I was going to address that later. But let me see what I can do

Reply:
Can this be a problem for Fusion resources?